zt_frontend/src/components/FileExplorer.vue

```diff
@@ -247,7 +229,19 @@ export default defineComponent({
       }
     };
 
+    const editableExtensions = [
```

Comment:
Thinking about this more, what is the error shown if the file isn't editable? There may be cases where people have custom file extensions that should be editable (just interpreted as a text file or something like that). If we just need to handle an error and say that the file isn't editable due to some caught error that is probably preferable

Reply:
Regarding dev mode i am sorry i missed on that while editing  i will update that rightnow

Reply:
For now in non editable files if the file is not editable , the edit option is not available in the menu itself , but if we want to handle it in a different way i will update it

Reply:
The issue is that this list of files may not actually be exhaustive. It would likely be better to just return an error if a user tries to edit a file that is not editable

Reply:
Understood i will update on file Editable Error

Reply:
Also i have solved  the dev mode and download issue

Reply:
I worked on eidtor changes and have updated accordingly, have also solve the build error and rebased the branch